sdk/clientcore/core/src/test/java/io/clientcore/core/implementation/http/rest/SimpleXmlSerializable.java

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package io.clientcore.core.implementation.http.rest;
+
+import io.clientcore.core.implementation.util.ImplUtils;
+import io.clientcore.core.serialization.xml.XmlReader;
+import io.clientcore.core.serialization.xml.XmlSerializable;
+import io.clientcore.core.serialization.xml.XmlToken;
+import io.clientcore.core.serialization.xml.XmlWriter;
+
+import javax.xml.stream.XMLStreamException;
+import java.util.Objects;
+
+/**
+ * Test class implementing {@link XmlSerializable}.
+ */
+public final class SimpleXmlSerializable implements XmlSerializable<SimpleXmlSerializable> {
```

Comment:
Simple or DefaultXmlSerializable provided by clientcore?

Reply:
This is a test class where we implement the XML streaming APIs from `io.clientcore.core.serializer.xml` to ensure XML support works in `RestProxy`.